src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-position: center;
+  flex-direction: row;
```

Comment:
flex-direction nothing do without display:flex please remove it. And then you add display: flex, flex-direction: row by default